### ISSUE TITLE
Cannot test TTkString against NoneType

### ISSUE THREAD

**Issue by nickandwolf** [ceccopierangiolieugenio/pyTermTk#109]
self.firstName is variable that starts None and become a string later.
It is later assigned a value by a LineEdit.

```
if self.firstName == None:

  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/TermTk/TTkCore/string.py", line 140, in __eq__
    def __eq__(self, other): return self._text == other if type(other) is str else self._text == other._text
AttributeError: 'NoneType' object has no attribute '_text'

```

**Comment by ceccopierangiolieugenio** [comment]
I need to fix it, in the meantime you can use
```python
if self.firstName is None:
```

**Comment by ceccopierangiolieugenio** [comment]
fixed in *0.20.0a4*
